Documentation/Settings.cfg

```diff
@@ -1,21 +1,62 @@
-[general]
-
-project = Surf
-release = master
-version = 3.3
+# More information about this file:
+# https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#settings-cfg
 
-[notify]
+[general]
 
-about_new_build = no
+project     = Surf
+version     = main
```

Comment:
nop, we are still on master

Reply:
yes, but docs.typo3.org already uses main only, see https://docs.typo3.org/other/typo3/surf/**main**/en-us/.

Reply:
so the branch does not need to match the version? that is really confusing

Reply:
It is just a convention that we always refer to "main", no matter if the branch is called master, main or latest or production. "main" is used in the version switch anyways on docs.typo3.org, so it might be confusing to the reader to have "master" in the release table row.

Reply:
ok